### PR TITLE
Added DLCmon data

### PR DESCRIPTION
**Pull request by RedzepR**
Added game data, image files and related data (evo stones) for all the SV dlc 1: teal mask pokemon

## Description

The SV dlc 1: teal mask added a few new pokemon. While not doing anything with the added data this helps anyone who wants to develope future content with these mons

## Motivation and Context

The data needs to be added sooner or later

## How Has This Been Tested?

Went into the game and caught the mons

## Types of changes

- new pokemon data

### CODE REVIEW

**Comment by Qwertypop04** [comment]
Munkidori, Okidogi and Fezandipiti are 100% male

**Comment by RedzepR** [comment]
> Munkidori, Okidogi and Fezandipiti are 100% male

Thanks for pointing out

File: src/modules/pokemons/PokemonList.ts [hunk 1]
```diff
@@ -30477,6 +30498,204 @@ export const pokemonList = createPokemonArray(
...
+            'attack': 120,
+            'specialAttack': 60,
+            'defense': 84,
+            'specialDefense': 96,
+            'speed': 110,
+        },
+        'eggCycles': 10,
+        'levelType': LevelType.slow,
+        'exp': 275,
+        'catchRate': 3,
```
Comment (Qwertypop04) [inline src/modules/pokemons/PokemonList.ts:30634]: Ogerpon's catch rate should be 5, rather than 3

**Review by Qwertypop04 — APPROVED** [review]
Double checked all the numbers, everything looks correct. Thanks for the PR!

**Pull request merged**